lib/fluent/system_config.rb

```diff
@@ -55,6 +55,8 @@ class SystemConfig
     config_section :log, required: false, init: true, multi: false do
       config_param :format, :enum, list: [:text, :json], default: :text
       config_param :time_format, :string, default: '%Y-%m-%d %H:%M:%S %z'
+      config_param :rotate_age, :string, default: nil
+      config_param :rotate_size, :integer, default: nil
```

Comment:
How about use `size` type instead of `integer`?
`size` type can handle `k`, `m`, `g` SI prefix.

Reply:
Using `size` seems reasonable, thanks!